core/src/main/java/org/mule/lifecycle/phases/ContainerManagedLifecyclePhase.java

```diff
@@ -9,7 +9,10 @@
 /**
  * A container-managed lifecycle phase is called by the container itself. This class acts as a marker to enable
  * tracking of phases even if Mule does not initiate the phase
+ *
+ * @deprecated as of 3.7.0 since it's no longer used
  */
+@Deprecated
 public class ContainerManagedLifecyclePhase extends DefaultLifecyclePhase
```

Comment:
Not sure what was this class used for, but if it was used for embedded deployments, then this must be fully tested